# Mavlink_mission config parameter

# 设备编号	/9-地面站
system_id: 1

# 串口端口/dev/ttyUSB*
uart_name_No: 1

# 串口波特率 /57600 /115200 /961200 ...
baudrate: 57600

# 1发送 0读取
case0r_1s: 2

#是否发送UDPdata
UDPdataflag: 0

#是否发送QGCommands
QGCflag: 1

#gs_id
gs_id: 1

#vehicleNumber
vehicleNumber: 40

#taskType
taskType: 1

#plane_state
plane_state: 2

#plane_group
plane_group: 1

#gapx
gapx: 5

#gapy
gapy: 5

